WooCommerce/UITestsFoundation/Screens/Products/ProductsScreen.swift

```diff
@@ -75,4 +75,15 @@ public final class ProductsScreen: ScreenObject {
         XCTAssertTrue(isLoaded)
         return self
     }
+
+    public func tapAddProduct() throws -> Self {
+        app.buttons["product-add-button"].tap()
+        return self
+    }
+
+    public func selectProductType(productType: String) throws -> SingleProductScreen {
+        app.staticTexts[productType].tap()
+
+        return try SingleProductScreen()
```

Comment:
Just noticing that there's a line break before `return`, while in two functions above there isn't:

```suggestion
        app.staticTexts[productType].tap()
        return try SingleProductScreen()
```